mungegithub/mungers/milestone-maintainer.go

```diff
@@ -279,6 +281,130 @@ func durationGreaterThanZero(name string, value time.Duration) error {
 	return nil
 }
 
+func dayPhrase(days int) string {
+	dayString := "days"
+	if days == 1 || days == -1 {
+		dayString = "day"
+	}
+	return fmt.Sprintf("%d %s", days, dayString)
+}
+
+func durationToMinDays(duration time.Duration) string {
+	days := int(math.Floor(duration.Hours() / 24))
+	return dayPhrase(days)
+}
+
+func durationToMaxDays(duration time.Duration) string {
+	days := int(math.Floor(duration.Hours() / 24))
+	return dayPhrase(days)
+}
+
+func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool) {
+	pr, ok := obj.GetPR()
+	if !ok {
+		return nil, ok
+	}
+
+	comments, ok := obj.ListReviewComments()
+	if !ok {
+		return nil, ok
+	}
+
+	lastHuman := pr.CreatedAt
+	for i := range comments {
+		comment := comments[i]
+		if comment.User == nil || comment.User.Login == nil || comment.CreatedAt == nil || comment.Body == nil {
+			continue
+		}
+		if obj.IsRobot(comment.User) || *comment.User.Login == jenkinsBotName {
+			continue
+		}
+		if lastHuman.Before(*comment.UpdatedAt) {
+			lastHuman = comment.UpdatedAt
+		}
+	}
+
+	return lastHuman, true
+}
+
+func findLastHumanIssueUpdate(obj *github.MungeObject) (*time.Time, bool) {
+	lastHuman := obj.Issue.CreatedAt
+
+	comments, ok := obj.ListComments()
+	if !ok {
+		return nil, ok
+	}
+
+	for i := range comments {
+		comment := comments[i]
+		if !validComment(comment) {
+			continue
+		}
+		if obj.IsRobot(comment.User) || jenkinsBotComment(comment) {
+			continue
+		}
+		if lastHuman.Before(*comment.UpdatedAt) {
+			lastHuman = comment.UpdatedAt
+		}
+	}
+
+	return lastHuman, true
+}
+
+func findLastInterestingEventUpdate(obj *github.MungeObject) (*time.Time, bool) {
+	lastInteresting := obj.Issue.CreatedAt
+
+	events, ok := obj.GetEvents()
+	if !ok {
+		return nil, ok
+	}
+
+	for i := range events {
+		event := events[i]
+		if event.Event == nil || *event.Event != "reopened" {
+			continue
+		}
+
+		if lastInteresting.Before(*event.CreatedAt) {
+			lastInteresting = event.CreatedAt
+		}
+	}
+
+	return lastInteresting, true
+}
+
+func findLastModificationTime(obj *github.MungeObject) (*time.Time, bool) {
```

Comment:
All the `findLast*` functions in this file except for this one (`findLastModificationTIme`) were only used by the close-stale munger and can now be deleted.